docs/migration-guide.md

```diff
@@ -2,11 +2,196 @@
 
 ## Contents
 
+- [Neutrino v8 to v9](#neutrino-v8-to-v9)
 - [Neutrino v7 to v8](#neutrino-v7-to-v8)
 - [Neutrino v6 to v7](#neutrino-v6-to-v7)
 - [Neutrino v5 to v6](#neutrino-v5-to-v6)
 - [Neutrino v4 to v5](#neutrino-v4-to-v5)
 
```

Comment:
Should we add a link to the issues section saying please file an issue if any problems found when upgrading that aren't mentioned etc / they need help?